treeherder/model/fixtures/repository.json

```diff
@@ -1413,5 +1413,31 @@
       "repository_group": 11,
       "description": "Fenix is not your parent's Android browser"
     }
+  },
+  {
+    "pk": 109,
+    "model": "model.repository",
+    "fields": {
+      "dvcs_type": "hg",
+      "name": "mozilla-esr68",
+      "url": "https://hg.mozilla.org/releases/mozilla-esr68",
+      "active_status": "active",
+      "codebase": "gecko",
+      "repository_group": 2,
+      "description": ""
+    }
+  },
+  {
+    "pk": 110,
+    "model": "model.repository",
+    "fields": {
+      "dvcs_type": "hg",
+      "name": "comm-esr68",
+      "url": "https://hg.mozilla.org/releases/comm-esr68",
+      "active_status": "onhold",
```

Comment:
Putting this as onhold, as the repository has not be created yet, and I'm not sure when it will be. Adding it here so that the repositories have consecutive keys.